src/doc/rustdoc/src/unstable-features.md

```diff
@@ -664,3 +664,22 @@ Similar to cargo `build.rustc-wrapper` option, this flag takes a `rustc` wrapper
 The first argument to the program will be the test builder program.
 
 This flag can be passed multiple times to nest wrappers.
+
+### `--extract-doctests`: outputs doctests in JSON format
+
+ * Tracking issue: [#134529](https://github.com/rust-lang/rust/issues/134529)
+
+When this flag is used, it outputs the doctests original source code alongside information
+such as:
+
+ * File where they are located.
+ * Line where they are located.
+ * Codeblock attributes (more information about this [here](./write-documentation/documentation-tests.html#attributes)).
+
+The output format is JSON.
+
+Using this flag looks like this:
+
+```bash
+$ rustdoc -Zunstable-options --extract-doctests src/lib.rs
```

Comment:
Why is this a CLI option and not an output format?

Reply:
Because I didn't think about it, it's SO MUCH better!